SensusUI/Inputs/LabelSliderInput.cs

```diff
@@ -0,0 +1,266 @@
+                            _incrementalValue = newIncrementalValue;
+                            _incrementalValueHasChanged = true;
+                            Complete = Value != null;
+                        }
+                    };
+
+                base.SetView(new StackLayout
+                    {
+                        Orientation = StackOrientation.Vertical,
+                        VerticalOptions = LayoutOptions.Start,
+                        Children =
+                            { 
+                                _sliderLabel,
+                                new StackLayout
+                                {
+                                    Orientation = StackOrientation.Horizontal,
+                                    HorizontalOptions = LayoutOptions.FillAndExpand,
+                                    Children = { _slider }
+                                },
+                                new StackLayout
+                                {
+                                    Orientation = StackOrientation.Horizontal,
+                                    HorizontalOptions = LayoutOptions.FillAndExpand,
+                                    Children = 
+                                        {
+                                            new Label
+                                            {
+                                                Text = _leftLabel,
+                                                HorizontalOptions = LayoutOptions.FillAndExpand,
+                                                FontSize = 15
+                                            },
+
+                                            new Label
+                                            {
+                                                Text = _rightLabel,
+                                                HorizontalOptions = LayoutOptions.End,
+                                                FontSize = 15
+                                            }
+                                        }
+                                }
+                            }
+                        });
+            }
+            else
+            {
+                // if the view was already initialized but is not enabled and has never been interacted with, there should be no tip text since the user can't do anything with the slider.
+                if (!Enabled && !_incrementalValueHasChanged)
+                {
+                    _slider.Value = _minimum;
+                    _sliderLabel.Text = GetLabelText(index) + "  No value selected.";
+                }            
+            }
+
+            return base.GetView(index);
+        }
+
+        private double GetIncrementalValue(double value)
+        {
+            return Math.Round(value / _increment) * _increment;
+        }
+
+        public override string ToString()
+        {
+            return base.ToString() + " -- " + _minimum + " to " + _maximum;
+        }
+    }
+}
+
```

Comment:
Why not just add the left and right label properties to the existing slider input? If one or both are non-empty, display the additional StackLayout below the slider; otherwise, the input is identical to the exiting slider, right?